plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/MavenPathParser.java

```diff
@@ -10,7 +10,7 @@
  * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
  * Eclipse Foundation. All other trademarks are the property of their respective owners.
  */
-package org.sonatype.nexus.repository.maven.internal;
+package org.sonatype.nexus.repository.maven;
```

Comment:
I think these bits should be publicly accessible (ie. accessible from another bundle), I guess stuff like Staging/CLM/HC would like to use bits from it (today it uses GavCalculator from NX2, that might be replaced by think `MavenPathParser`)